### ISSUE TITLE
Add a rule on scm > tag element in the pom.xml (maven-release plugin)

### ISSUE THREAD

**Issue by seblatre** [cecom/pomutils#11]
When using maven-release plugin, the goals release;branch and release:prepare are both modifiying the `scm > tag` element in the pom.xml.
* maven:prepare set `scm > tag` to the new Git branch 
![maven-branch](https://user-images.githubusercontent.com/7345463/48420943-cc464e80-e75b-11e8-9f2a-0924982b218e.png)
* maven:prepare set `scm > tag` to the Git tag generated during the release process then put it back to head in the next commit for development
![maven-release](https://user-images.githubusercontent.com/7345463/48420944-cc464e80-e75b-11e8-99ff-827cd20826a1.png)

Therefore, there can be conflicts on this item when merging pom.xml. A new rule could be added to manage such behavior automatically.

**Comment by seblatre** [comment]
I have already coded such implementation of the rule, can I push it to the project?

**Comment by cecom** [comment]
Hi,

Sure. Send me a pull request and i will have a look and merge it.

Greetings,
Sven

Von meinem iPhone gesendet

**Comment by seblatre** [comment]
@cecom: Unfortunately, I'm unable to push my branch to the repo. It denies me the right.
```
xxx (seblatre/scmtag)
$ git remote -v
github  git@github.com:cecom/pomutils.git (fetch)
github  git@github.com:cecom/pomutils.git (push)

xxx (seblatre/scmtag)
$ git push --set-upstream github seblatre/scmtag
Warning: Permanently added the RSA host key for IP address '140.82.118.4' to the list of known hosts.
ERROR: Permission to cecom/pomutils.git denied to seblatre.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by cecom** [comment]
You have to create a fork in your github account and from there you can send me a pull request from your fork.

Von meinem iPhone gesendet

**Comment by seblatre** [comment]
Ok, done ^^
Sorry for my misunderstanding of github.

**Comment by cecom** [comment]
NoProb :-) 

Can you plz add test for your change? Its easy, have a look at the other tests. And plz send the pull request with target develop not master.

Greetings,
Sven

Von meinem iPhone gesendet

**Comment by seblatre** [comment]
Hello @cecom,
I finally got time to implement unit tests. Can you have a look and confirm that it's ok for you?

**Comment by cecom** [comment]
Hello @seblatre,

it is now released in 1.6.0. We forgot to adjust the README in the pull requests. I added that. Thanks for your rule :)
